### ISSUE TITLE
Issue with FineGym99 Features File Shape Discrepancy

### ISSUE THREAD

**Issue by ImtiazUlHassan** [lizhi1104/HAAN#5]
I encountered an issue with the `FineGym99` features file provided on the cloud. Specifically, when reading the `FineGym_i3d_fps24.npy` NumPy file, its shape is `(4968,)`, which does not align with the expected shape based on the original dataset.

According to the original dataset:
- The training set should have `26320` samples.
- The validation set should have `8521` samples.

However, the provided file contains significantly fewer samples than expected. Could you clarify whether the data has been processed differently or if there is an issue with the provided file?

**Steps to Reproduce:**
1. Download the `FineGym_i3d_fps24.npy` file from the cloud.
2. Load it using NumPy:
   ```python
   import numpy as np
   data = np.load("FineGym_i3d_fps24.npy", allow_pickle=True)
   print(data.shape)
   ```
3. Observe the output shape `(4968,)`.

**Expected Behavior:**
The file should contain features for the full dataset, matching the training (`26320`) and validation (`8521`) samples.

**Additional Context:**
Please confirm whether the dataset has been filtered or modified before uploading, or if there was an issue in the data processing pipeline.

Thank you!

**Comment by taoerman** [comment]
Thanks for your question!

I believe 26320 + 8521 is the number of sub-actions. We are proposing to detect sub-actions within actions, each action may contain multiple sub-actions. 4968 is the total number of actions, in which 26320 + 8521 sub-actions are present.

**Comment by lizhi1104** [comment]
Thanks for your question!

I believe 26320 + 8521 is the number of sub-actions. We are proposing to detect sub-actions within actions, each action may contain multiple sub-actions. 4968 is the number of actions, in which 26320 + 8521 sub-actions are present.
